dlt/extract/extractors.py

```diff
@@ -43,6 +46,7 @@ class ExtractorConfiguration(BaseConfiguration):
         _caps: Optional[DestinationCapabilitiesContext] = None
 
     @with_config(spec=ExtractorConfiguration)
+    @with_plugins()
```

Comment:
you do not need this decorator, `with_config` will do that for you. there's a special config provider `ContextProvider` that will inject the `PluginContext` into `_plugins`. exactly like `_caps`